### PR TITLE
Refactor blockquotes / pull quotes

### LINKED ISSUE #394: Audit blockquotes pattern
## Context

There are currently two different types of blockquote - could these be distilled into one pattern that is sufficiently responsive to satisfy both use cases?

* Any styling applied directly on `<blockquote>` should be moved base layer.
* This pattern be renamed "Pull quote"
* Pull quotes should always fill 100% of it's container and any styling should be controlled via the grid tools.

## Audit Checklist

- [x] Browser testing
- [x] Accessibility testing
- [x] Sass linting
- [x] Update documentation
- [x] Convert sizing units to rems

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

- Split out base blockquote styling from pull quote pattern
- Deprecate row quotes

## QA

Pull down changes, sanity check code and check pattern in vf.io

## Details

Fixes #394

### CODE REVIEW

File: scss/_base_blockquotes.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      line-height: 1.5;
+    }
+
+    cite {
+      width: 100%;
+      margin-left: 3rem;
+    }
+
+    &::before {
+      content: '\201C'; /*Unicode for Left Double Quote*/
```
Comment (anthonydillon) [inline scss/_base_blockquotes.scss]: Where is the right double quote? Shouldn't there be an `:after`

File: scss/_base_blockquotes.scss [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      font-size: 60px;
```
Comment (anthonydillon) [inline scss/_base_blockquotes.scss]: rem?

File: scss/_patterns_pull-quotes.scss [hunk 3]
```diff
@@ -0,0 +1,15 @@
+// Pull quote styling
+@mixin vf-p-pull-quotes {
+
+  .p-pull-quote {
+
```
Comment (anthonydillon) [inline scss/_patterns_pull-quotes.scss]: extra white space.

**Comment by anthonydillon** [comment]
Code has a few comments and the QA doesn't seem right:
blockquotes vanilla framework

**Comment by barrymcgee** [comment]
@anthonydillon Thanks - ready for re-review.

**Comment by anthonydillon** [comment]
LGTM :+1:

**Pull request merged**